docs/populating-configuration.md

```diff
@@ -374,3 +374,11 @@ previously created.
 
 Additionally, make sure to set the Sentry URL endpoint and Sentry project when
 starting the Fleet Manager server. See [Sentry-related CLI flags in Fleet Manager](./feature-flags.md#sentry)
+
+## Configure data plane observability & remote write
```

Comment:
Do you think this should belong in `Setup the Observability stack secrets` instead?

Reply:
Yes, that makes sense. Looking at that section, it needs to be updated anyway because it doesn't yet describe how to set up the proxy. I'd prefer to do this in a different PR.

Reply:
> Yes, that makes sense. Looking at that section, it needs to be updated anyway because it doesn't yet describe how to set up the proxy. I'd prefer to do this in a different PR.

I already have some local branch describing how to setup the proxy. You can add it in here and I will join everything